NetKVM/wlh/ParaNdis6-Oid.cpp

```diff
@@ -1243,7 +1243,11 @@ void ParaNdis6_Fill620PowerCapabilities(PNDIS_PM_CAPABILITIES pPower620Caps)
     pPower620Caps->Header.Type = NDIS_OBJECT_TYPE_DEFAULT;
     pPower620Caps->Header.Revision = NDIS_PM_CAPABILITIES_REVISION_1;
     pPower620Caps->Header.Size = NDIS_SIZEOF_NDIS_PM_CAPABILITIES_REVISION_1;
-
+#if NDIS_SUPPORT_NDIS650
```

Comment:
Shouldn't we use here also "if (CheckNdisVersion())" construct here?
The function itself is under 6.2 define and revision 2 supported form 6.3.

Reply:
Same as here: https://github.com/virtio-win/kvm-guest-drivers-windows/pull/450/commits/3b32c43e31eebfd88cf213d05121fdf85d73e35b

Reply:
Run-time NDIS version currently is:
6.30 for any system with NDIS < 6.80
6.80 for any system with NDIS >= 6.80
So CheckNdisVersion currently allows to distinguish between 6.80 and 6.30.
For <= Win 8.1 the code here stays fully backward compatible (intentionally).
For all the Win 10 we indicate revision 2

Reply:
Maybe worth adding a comment.